.github/actions/setup-environment/action.yml

```diff
@@ -248,6 +251,23 @@ runs:
         if [ "$ENVIRONMENT_NAME" = "Branch" ]; then
           PR_NUMBER=${{ github.event.number || github.event.inputs.pr-number || inputs.pr-number }}
           HAS_CONTENTFUL_LABEL=${{ contains(github.event.pull_request.labels.*.name, 'contentful') }}
+          AUTH0_LABEL=${{contains(github.event.pull_request.labels.*.name, 'use-pr-auth0-tenant') }}
+          echo "crn-auth0-label=$AUTH0_LABEL" >> $GITHUB_OUTPUT
```

Comment:
is this crn-only label? doesn't align with what is inside the condition below (looks like we set the gp2 client IDs of the PR tennant for when the label is on)

Reply:
The label will give you an auth0 PR env for both apps.